velox/common/time/tests/CpuWallTimerTest.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+#include <thread>
+
+#include "velox/common/time/CpuWallTimer.h"
+
+using namespace facebook::velox;
+
+namespace facebook::velox::test {
+
+class CpuWallTimerTest : public testing::Test {
+ protected:
+  static void workAndSleep(
+      size_t workIterations,
+      std::chrono::nanoseconds sleepDuration) {
+    size_t n{12};
+    for (size_t i = 0; i < workIterations + 10; ++i) {
+      n += 5 + i * 10;
```

Comment:
What's the logic here?

Reply:
Nothing special, just to use some cpu for arithmetics.